### PR TITLE
Add the cgive command

### PR DESCRIPTION
**Pull request by haykam821**
This command allows creative mode players to give a stack of items to themselves with arbitrary NBT.

### CODE REVIEW

File: src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        dispatcher.register(literal("cgive")
+            .then(argument("item", itemStack())
+            .executes(ctx -> give(ctx.getSource(), getItemStackArgument(ctx, "item"), 1))
+                .then(argument("count", integer(1))
+                .executes(ctx -> {
+                    return give(ctx.getSource(), getItemStackArgument(ctx, "item"), getInteger(ctx, "count"));
+                }))));
+    }
+
+    private static int give(ServerCommandSource source, ItemStackArgument itemArgument, int count) throws CommandSyntaxException {
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java:33]: You seem to be using a mixture of tabs and spaces in this method. Please stick to spaces.

File: src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("cgive");
+
+        dispatcher.register(literal("cgive")
+            .then(argument("item", itemStack())
+            .executes(ctx -> give(ctx.getSource(), getItemStackArgument(ctx, "item"), 1))
+                .then(argument("count", integer(1))
+                .executes(ctx -> {
+                    return give(ctx.getSource(), getItemStackArgument(ctx, "item"), getInteger(ctx, "count"));
```
Comment (Earthcomputer) [inline src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java]: Could you make this an expression lambda?

**Review by Earthcomputer — CHANGES_REQUESTED** [review]

File: src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java (hunk 1 above)
Comment (haykam821) [reply src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java:33]: Fixed in f5e8993ee9f5f77b435f85fd866d82f746566d9d

File: src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java (hunk 2 above)
Comment (haykam821) [reply src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java]: Fixed in ebfb51c976bc875e30699603732a1660b12d8d94

**Review by Earthcomputer — APPROVED** [review]

**Pull request merged**